packages/block-editor/src/components/block-list/block-list-item.native.js

```diff
@@ -16,14 +16,82 @@ import { ReadableContentView } from '@wordpress/components';
  */
 import BlockListBlock from './block';
 import BlockInsertionPoint from './insertion-point';
+import styles from './block-list-item.native.scss';
+
+const BREAKPOINTS = {
+	medium: 700,
```

Comment:
Could you please explain why the `medium` value is different than `medium` value in `packages/components/src/mobile/readable-content-view/index.native.js`?

Reply:
Good catch, I'm updating this to use the `medium` value from the new utils. Thanks!